preamble.tex

```diff
@@ -1,10 +1,49 @@
-%
-% --- inline annotations
-%
+%%
+%% fine-tune paragraph spacing
+%%
+% \renewcommand{\paragraph}[1]{\vspace{.5em}\noindent\textbf{#1.}}
+
+%%
+%% globally adjusts space between figure and caption
+%%
+% \setlength{\abovecaptionskip}{.5em}
+
+
+%%
+%% Allows "the use of \paper to refer to the project name"
+%% with automatic management of space at the end of the word
+%%
+% \usepackage{xspace}
+% \newcommand{\paper}{ProjectName\xspace}
+
+%%
+%% Commonly used math definitions
+%%
+% \DeclareMathOperator*{\argmin}{arg\,min}
+% \DeclareMathOperator*{\argmax}{arg\,max}
+
+%%
+%% Commonly colors
+%%
+% \definecolor{turquoise}{cmyk}{0.65,0,0.1,0.3}
+% \definecolor{purple}{rgb}{0.65,0,0.65}
+% \definecolor{dark_green}{rgb}{0, 0.5, 0}
+% \definecolor{orange}{rgb}{0.8, 0.6, 0.2}
+% \definecolor{red}{rgb}{0.8, 0.2, 0.2}
```

Comment:
Why override pre-defined colours?

Reply:
where are they pre-defined? These don't cause conflicts in compile

Reply:
The "xcolor" package predefines some basic colors, including red, purple, orange, pink, and teal. We should override these defaults.

![image](https://github.com/user-attachments/assets/6d45b9e4-7704-44b2-b70e-21e08e3e39c8)

Source: [Named colors provided by the xcolor package](https://www.overleaf.com/learn/latex/Using_colors_in_LaTeX#Named_colors_provided_by_the_xcolor_package)

Reply:
yah I think you are right, this was left for legacy reason, but indeed, better to leave a pointer where to get pre-defined colors